cryptohash-sha512.cabal

```diff
@@ -63,9 +66,9 @@ test-suite test-sha512
 
                    , base16-bytestring >= 0.1.1  && < 0.2
                    , SHA               >= 1.6.4  && < 1.7
-                   , tasty             == 0.11.*
-                   , tasty-quickcheck  == 0.8.*
-                   , tasty-hunit       == 0.9.*
+                   , tasty             >= 0.11   && < 1.1
```

Comment:
As I comented in the somewhat redundant #2 already:

> strictly speaking, this ought to be `== 0.11.* || == 0.12.* || == 1.0.*` assuming you can confirm that it works correctly with all 3 major versions.

Reply:
Made that way, test run successfully with all three majors - tried locally.